extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/FilteredShowObjectsExec.scala

```diff
@@ -47,7 +48,7 @@ case class FilteredShowNamespaceExec(delegated: SparkPlan) extends FilteredShowO
     val database = r.getString(0)
     val resource = AccessResource(ObjectType.DATABASE, database, null, null)
     val request = AccessRequest(resource, ugi, OperationType.SHOWDATABASES, AccessType.USE)
-    val result = SparkRangerAdminPlugin.isAccessAllowed(request)
+    val result = getOrCreateRangerPlugin().isAccessAllowed(request)
```

Comment:
which service stores privileges for all catalogs if we want a filtered list of catalogs

Reply:
No, this PR is not evolved for storing or checking users' privileges of catalog-level resources.
As for this line of the method, no action changed and it's filtering the database-level resources as in Hive service def, as the namespace-level in Spark concept.